### DISCUSSION TITLE
Clear grid selection

### DISCUSSION THREAD

**Discussion by bluezeus** [vikramlearning/blazorbootstrap#629] [Q&A]
Is it possible to clear the selection of a grid row?

**Comment by blazejjanus** [comment]
As far as I know there is no public method allowing this, so you can only achieve it via refreshing data `await grid.RefreshDataAsync();`.

I got similar problem when I used grid selection for bulk actions on some data. some of the calls like deleting items or changing it's data that were displayed in grid in fact needed refreshing data, but some like updating properties that are not displayed didn't and made API call that was not needed. To avoid it I introduced local backup that if created was taken instead of fetching API in DataProvider.
